src/jit/target.h

```diff
@@ -387,7 +387,13 @@ typedef unsigned short          regPairNoSmall; // arm: need 12 bits
   #define FEATURE_SET_FLAGS        0       // Set to true to force the JIT to mark the trees with GTF_SET_FLAGS when the flags need to be set
   #define FEATURE_MULTIREG_ARGS_OR_RET  0  // Support for passing and/or returning single values in more than one register
   #define FEATURE_MULTIREG_ARGS         0  // Support for passing a single argument in more than one register  
+
+#ifdef LEGACY_BACKEND
   #define FEATURE_MULTIREG_RET          0  // Support for returning a single value in more than one register  
+#else
+  #define FEATURE_MULTIREG_RET          1  // Support for returning a single value in more than one register  
+#endif
```

Comment:
If either FEATURE_MULTIREG_RET  or FEATURE_MULTIREG_ARGS  is enabled (by setting them to 1) then the value of FEATURE_MULTIREG_ARGS_OR_RET  must also be set to 1.

Probably best to expand this #ifdef on LEGACY backend to cover all three of these value: